libs/media-player/src/lib/transcript-line-presenter.tsx

```diff
@@ -0,0 +1,17 @@
+import { Typography } from '@mui/material';
+
+interface TranscriptLinePresenterProps {
```

Comment:
Sounds good, I'd need some more guidance to follow up on this.  Also, this touches on the modelling for the video player.  Do we have one more conventional video player along the lines of Youtube that "plays" lines of transcript as subtitles according to timecode as they watch?  and then have separate modelling for the interactive transcript tool?  This tool could auto scroll through the transcript in a div underneath as the video plays and highlight the line currently playing.  That way it could keep the context of the transcription as you suggest.